language_detector/main.py

```diff
@@ -1,8 +1,9 @@
-# -*- coding: utf-8 -*-
-
-"""This is the entry point of the program."""
-
 def detect_language(text, languages):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    counts = { lang['name']:0 for lang in languages }
+    words = text.split()
+    for w in words:
```

Comment:
Nice, elegant solution. Since you don't really use the words you could skip assigning the variable and just do it directly via:
`for word in text.split():`
Good job!